dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/JdbcPropBuilder.java

```diff
@@ -77,7 +75,11 @@ public JdbcPropBuilder prop(@Nonnull String prop, @Nonnull String val) {
     }
 
     private void addProp(String prop, String val) {
-        props.add(prop + punctuations.charAt(1) + URLEncoder.encode(val, UTF_8));
+        try {
+            props.add(prop + punctuations.charAt(1) + URLEncoder.encode(val, "UTF-8"));
+        } catch (UnsupportedEncodingException e) {
+            e.printStackTrace();
```

Comment:
This should go to a logger at the very least. I wonder how the InternalError above sneaked past review!